pilot/cmd/pilot-agent/status/util/stats.go

```diff
@@ -29,7 +29,8 @@ const (
 	statLdsRejected  = "listener_manager.lds.update_rejected"
 	statsLdsSuccess  = "listener_manager.lds.update_success"
 	statServerState  = "server.state"
-	updateStatsRegex = "^(cluster_manager.cds|listener_manager.lds).(update_success|update_rejected)$"
+	statsSdsSuccess  = "listener.0.0.0.0_15006.server_ssl_socket_factory.ssl_context_update_by_sds"
```

Comment:
I didn't think we expose this stat, do we?

Reply:
We expose it on 15000/stats